### PR TITLE
Update Receptionist.tex

### PR DESCRIPTION
**Pull request by myszzka96**
On the receptionist.txt task. 

Changed the doorbell to knock on the door. Cleared out some writing in the text for easier-to-understand descriptions. No changes to the task goal itself.

Changes proposed in this pull request:
- change the doorbell to knock on the door
- clarify descriptions

### CODE REVIEW

File: tasks/Receptionist.tex [hunk 1]
```diff
@@ -46,8 +45,7 @@ \subsection*{Procedure}
 	
 	\item \textbf{Describing the First Guest:} Naming at least 4 characteristics of the first guest, i.e., color of clothes, color of hair, gender, and age, earns bonus points.
 	\item \textbf{Looking at person/direction of navigation:} During verbal interactions and descriptions of people, robot 
-	looks at the conversational partner. Robot can point at the person being introduce/described or alternate gaze between two people. During navigation robot looks in the direction where it is going. Persistently gazing towards unrelated 
-	person or incorrect direction while moving during the task deducts points. 
+	looks at the conversational partner. Robot can point at the person being introduce/described or alternate gaze between host and guest(s) during introductions. During navigation robot looks in the direction where it is going. Persistently gazing away from direction of navigation and/or away from the interaction partner during the task deducts points.
```
Comment (johaq) [inline tasks/Receptionist.tex:48]: *introduced

Otherwise looks good to me

**Review by johaq — CHANGES_REQUESTED** [review]

File: tasks/Receptionist.tex [hunk 2]
```diff
@@ -78,10 +76,10 @@ \subsubsection*{To Referee}
 
 \subsubsection*{To OC}
 
-During setup day:
-\begin{itemize}
-	\item Provide the dorbell sound.
-\end{itemize}
+% During setup day:
```
Comment (PetervDooren) [inline tasks/Receptionist.tex:79]: I would fully remove this instead of commenting it out

File: tasks/Receptionist.tex [hunk 3]
```diff
@@ -30,8 +30,7 @@ \subsection*{Setup}
 	\begin{itemize}
 	\item \textbf{Host:} The host's name and favorite drink will be announced before the test. The host is already sitting in the living room.
 
-	\item \textbf{Guests:} Both guests have a name and favorite drink. An arriving guest will either step in front of the robot or ring the bell if the door is closed. Guests have to be guided to the living room to be introduced. Each of the guest will arrive separately. The sound of the bell will be provided to teams
-	during setup period before competition.
+	\item \textbf{Guests:} Both guests have a name and favorite drink. An arriving guest will either step in front of the robot or knock on the door if the door is closed. Guests have to be guided to the living room to be introduced. Each of the guest will arrive separately.
```
Comment (PetervDooren) [inline tasks/Receptionist.tex:33]: For future reference. What is the reason to switch to knocking?

Comment (johaq) [reply tasks/Receptionist.tex:33]: We don't rely on the sound setup on site to work consistently for every test run

**Comment by MatthijsBurgh** [comment]
The `2023_test` branch should not be used for this. I used it to test the new release procedure. I thought I deleted it. Just like I deleted the `2023` branch after merging it back to master.

So when you decide to again update the rulebook, while @johaq already announced the rulebook is final, I will prepare a new 2023 branch to merge to. (But I don't think you should update it anymore). But let me know, when I should prepare a new branch.

**Comment by myszzka96** [comment]
Sorry for the wrong branch - I can change it to go to master. Also, I am fine with not integrating those changes till next year. I just remembered this was addressed during the last RoboCup in Thailand as we did not do the doorbell. But if we do not want to make more changes before July, which is fair given the timeline, it can be postponed.

**Comment by PetervDooren** [comment]
@myszzka96 Do you still want to merge this or should we close this PR. There seems to be consensus on this PR save for a few minor changes.

**Pull request closed without merging**